hack/update-all.sh

```diff
@@ -17,17 +18,16 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-# Runs all hack/update-*.sh scripts
+# NB: This script requires bazel, and is no longer supported since we no longer support bazel
+# It's preserved for now but might be removed in the future
 
 
 hack=$(dirname "${BASH_SOURCE[0]}")
 
+echo -e "\033[0;33mThis script is preserved for legacy reasons, and as such will also update bazel
+You shouldn't need to run this script or install bazel for normal development.
+Use 'make update-all' to do everything this script does without touching bazel\033[0m"
```

Comment:
I felt like removing `update-bazel` here would be a more disruptive change for people who use these `hack` scripts due to muscle memory, so I kept it here but added `make update-all` for more general use.